c10/util/StringUtil.h

```diff
@@ -181,11 +176,15 @@ inline void printQuotedString(std::ostream& stmt, const string_view str) {
         } else {
           // C++ io has stateful formatting settings. Messing with
           // them is probably worse than doing this manually.
+          // NOLINTNEXTLINE(*c-arrays*)
```

Comment:
Why can't this be a C++ array? Seems harmless and better than silencing the lint?

Reply:
This is template specification for c arrays.

Reply:
I am not sure whether std::array<char,3> can be printed by output stream like a string. It's better to keep the current code and replace it by libfmt in the future.